src/.editorconfig

```diff
@@ -166,7 +166,7 @@ dotnet_diagnostic.CA2002.severity = warning
 dotnet_diagnostic.CA2003.severity = none
 dotnet_diagnostic.CA2004.severity = none
 dotnet_diagnostic.CA2006.severity = none
-dotnet_diagnostic.CA2007.severity = error
+dotnet_diagnostic.CA2007.severity = none
```

Comment:
CA2007 is kind of mandatory for library publishing (at least in warning severity) because it forces the use of `.ConfigureAwait(false)` on async calls. It's a must have for libraries that target .NET Standard 2.0 and could be used by apps running on the .NET Framework (like 4.6.2 or 4.7)